src/main/java/org/example/controller/DeveloperController.java

```diff
@@ -1,50 +1,63 @@
 package org.example.controller;
 
-import org.example.constant.ContextTypes;
-import org.example.constant.ResponseCodes;
+import org.example.constant.ContextType;
+import org.example.constant.JspPage;
+import org.example.constant.ResponseCode;
 import org.example.model.Arguments;
 import org.example.model.Developer;
-import org.example.model.Response;
+import org.example.model.JspAttribute;
+import org.example.model.JspAttributes;
 import org.example.service.DeveloperService;
 import org.example.service.JsonService;
-import org.example.service.TableService;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class DeveloperController {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DeveloperController.class);
     private final DeveloperService developerService = DeveloperService.getInstance();
-    private final TableService tableService = TableService.getInstance();
     private final JsonService jsonService = JsonService.getInstance();
 
 
     private static DeveloperController instance;
 
-    public Response getTableType(Arguments arguments) {
+    public JspAttributes getTableType(Arguments arguments) {
         List<Developer> developers = developerService.getDevelopers();
```

Comment:
Add logging about SQL execution result

Reply:
I had this logging in class 'DeveloperService'